tasks/setup.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Add system group
+  group:
+    name: "{{ database_migrations_system_user }}"
+    state: present
+
+- name: Add system user
+  user:
+    name: "{{ database_migrations_system_group }}"
+    shell: /bin/bash
+    group: "{{ database_migrations_system_user }}"
+    append: true
+    create_home: true
+
+- name: Install system-wide dependencies
+  apt:
+    name: "{{ database_migrations_system_dependencies }}"
+    state: present
+    update_cache: true
+    cache_valid_time: "600"
+  when:
+    - database_migrations_system_dependencies | length > 0
+
+- name: Ensure required directories are present
+  file:
+    state: directory
+    owner: "{{ database_migrations_system_user }}"
+    group: "{{ database_migrations_system_group }}"
+    path: "{{ item }}"
+  when:
+    - item is defined
+    - item != None
+  with_items:
+    - "{{ database_migrations_log_path }}"
+
+- name: Copy pgpass credentials
+  template:
+    src: "pgpass.j2"
+    dest: "{{ database_migrations_home }}/.pgpass"
+    owner: "{{ database_migrations_system_user }}"
+    group: "{{ database_migrations_system_group }}"
+    mode: "600"
+  when:
+    - database_migrations_db_connections.values() | length > 0
+    - database_migrations_create_pgpass | bool
```

Comment:
Why is this required given we have `db/setup.yml` below?

Reply:
@ukanga I was thinking that one may want to run/rerun this role and not touch any existing .pgpass giles